techmatters/hrm
# CHI-1108: GET /resource

Review after #284 is merged

## Description

* Adds a GET /resource endpoint
* Adds Resources table
* Adds an 'updatedSequence'
* Separates resources schema creation into separate script, so is now self contained within the resources module
* renames 'user' to 'twilioUser'

### Checklist

### Related Issues
CHI-1108

### Verification steps

* Ensure a resourceBaseURL is set in the service configuration for the dev account.
* Deploy this build to HRM dev
* Add a resource with an ID matching the hardcoded stub ID value currently set in Flex for the resources
* Open the resources view

It should load the resource data from the DB
